packages/core/handle/handle.js

```diff
@@ -131,9 +131,11 @@ const makeHandler = (handlers) => {
 };
 
 // Loose check to determine if obj is component-ish if it has both props and context members
-const hasPropsAndContext = (obj) => {
-	return obj && obj.hasOwnProperty && obj.hasOwnProperty('props') && obj.hasOwnProperty('context');
-};
+const hasPropsAndContext = (obj) => (
+	obj &&
+		Object.prototype.hasOwnProperty.call(obj, 'props') &&
+		Object.prototype.hasOwnProperty.call(obj, 'context')
```

Comment:
These two lines shouldn't need to be indented further, should they?

Reply:
I believe that's a product of linting auuto-fix, so I suppose they do. And it makes sense to indent next-line continuationations of expressions, right?

Reply:
Hmm, no, we didn't before, AFAIK.  It doesn't look like it updated similar lines in sandstone.

Reply:
I guess we don't have anything else exactly like this.  I guess it's fine!